layers/+distributions/spacemacs-base/funcs.el

```diff
@@ -979,3 +979,17 @@ a split-side entry, its value must be usable as the SIDE argument for
                                             text-scale-mode-amount) 1)
                                   (if (car (window-margins))
                                       (car (window-margins)) 1)))))
+
+(defun spacemacs/linum-enabled-p ()
```

Comment:
Function name is not accurate. Better name: `spacemacs/enable-linum-p` (it's "enable" without a "d", not a typo)